packages/connect-thegraph-template/subgraph/src/aragon/mappings/MiniMeToken.ts

```diff
@@ -0,0 +1,9 @@
+import { ClaimedTokens as ClaimedTokensEvent, NewCloneToken, Approval } from '../../../generated/templates/MiniMeToken/MiniMeToken'
+import { Transfer as TransferEvent } from '../../../generated/templates/MiniMeToken/MiniMeToken'
+import { NewCloneToken as NewCloneTokenEvent } from '../../../generated/templates/MiniMeToken/MiniMeToken'
+import { Approval as ApprovalEvent } from '../../../generated/templates/MiniMeToken/MiniMeToken'
+
+export function handleClaimedTokens(event: ClaimedTokensEvent): void {}
+export function handleTransfer(event: TransferEvent): void {}
+export function handleNewCloneToken(event: NewCloneTokenEvent): void {}
+export function handleApproval(event: ApprovalEvent): void {}
```

Comment:
Don't we have any way to handle these tokens events (and don't we have to handle them)?

Reply:
What do you mean? The way to handle those events is to enter some code on those functions. And no, you're not obligated to handle them.

Reply:
Gotcha; I wanted to know if it was best to handle these, rather than leaving them unhandled by default.

Reply:
Yeah I understand your concern. It wouldnt make sense to handle them in the template and in the Voting example that the template presents. We could comment out the code, but we would also need to do that in the yaml data sources, which would be awkward if the user wanted to enable them. That's why I decided to compromise with just leaving them empty. You can always optimize a subgraph by commenting them out in its implementation.